pkg/controllers/backup/controller.go

```diff
@@ -108,11 +108,14 @@ func (h *handler) OnBackupChange(_ string, backup *v1.Backup) (*v1.Backup, error
 		return h.setReconcilingCondition(backup, err)
 	}
 
+	if _, err = h.backups.UpdateStatus(backup); err != nil {
+		return h.setReconcilingCondition(backup, err)
+	}
+
 	if backup.Status.LastSnapshotTS != "" {
 		if backup.Spec.Schedule == "" { // one-time backup
```

Comment:
For instance, if we know that validated backups get saved first, then here the logic can be:
```suggestion
		if backup.Status.BackupType == v1.OneTimeBackupType { // one-time backup
```

Similar to the logic update done in `backupIsSingularAndComplete`